# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bacterium/bacterium_gui.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-11-09 13:50:16.061560752 +0100
+++ ./Makefile	2003-11-09 13:51:45.827914192 +0100
@@ -7,6 +7,10 @@
 
 all: bacterium_ffd bacterium_gui
 
+# place -lncurses after bacterium_ffd.c
+bacterium_ffd: bacterium_ffd.c
+	$(CC) -Wall bacterium_ffd.c -lncurses -o bacterium_ffd
+
 bacterium_gui: bacterium_gui.sh
 	sed 's,./\(bacterium_ffd\),\1,' < bacterium_gui.sh > bacterium_gui
 	chmod +x bacterium_gui
